pt-BR/mail/chrome/messenger/addressbook/abMainWindow.dtd

```diff
@@ -200,15 +200,15 @@
 <!ENTITY _PhoneticName.accesskey                        "f">
 
 <!-- Results pane context menu -->
-<!ENTITY composeEmail.label                             "Compose Email To">
+<!ENTITY composeEmail.label                             "Criar email para">
 <!ENTITY composeEmail.accesskey                         "C">
-<!ENTITY deleteAddrBookCard.label                       "Delete">
+<!ENTITY deleteAddrBookCard.label                       "Excluir">
 <!ENTITY deleteAddrBookCard.accesskey                   "D">
-<!ENTITY newAddrBookCard.label                          "New Card...">
+<!ENTITY newAddrBookCard.label                          "Novo cartão...">
 <!ENTITY newAddrBookCard.accesskey                      "N">
-<!ENTITY newAddrBookMailingList.label                   "New Mailing List...">
+<!ENTITY newAddrBookMailingList.label                   "Criar nova lista de emails...">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY newAddrBookMailingList.label                   "Criar nova lista de destinatários...">